File: CS/Exercises/Exercise2/Runner.cs
namespace Exercises.Exercise2;

class Runner
{
  public static void RunMe()
  {
    var persons = new AntaDataBuffer<PersonRow>();

    //// Commentaar opgave 1
    //var personRow = new PersonRow();
    //personRow.Id.Value = Guid.NewGuid();
    //personRow.FirstName.Value = "Larry";
    //personRow.LastName.Value = "Page";
    //persons.Add(personRow);

    //var myGuid = Guid.NewGuid();

    //personRow = new PersonRow();
    //personRow.Id.Value = myGuid;
    //personRow.FirstName.Value = "Sergey";
    //personRow.LastName.Value = "Brin";
    //persons.Add(personRow);

    //Console.WriteLine("List:");

    //foreach (var person in persons)
    //{
    //  Console.WriteLine($"{person.FirstName.Value} {person.LastName.Value}");
    //}

    //// Commentaar opgave 2
    //Console.WriteLine();
    //Console.WriteLine("Indexer:");
    //Console.WriteLine($"{persons[myGuid].FirstName.Value} {persons[myGuid].LastName.Value}");
  }
}

File: VB/Exercises/Exercise2/Runner.vb
Namespace Exercises.Exercise2
  Class Runner
    Shared Sub RunMe()
      Dim persons = New AntaDataBuffer(Of PersonRow)

      ''Commentaar opgave 1
      'Dim personRow = New PersonRow()
      'personRow.Id.Value = Guid.NewGuid()
      'personRow.FirstName.Value = "Larry"
      'personRow.LastName.Value = "Page"
      'persons.Add(personRow)

      'Dim myGuid = Guid.NewGuid()

      'personRow = New PersonRow()
      'personRow.Id.Value = myGuid
      'personRow.FirstName.Value = "Sergey"
      'personRow.LastName.Value = "Brin"
      'persons.Add(personRow)

      'Console.WriteLine("List:")

      'For Each person In persons
      '  Console.WriteLine("{0} {1}", person.FirstName.Value, person.LastName.Value)
      'Next

      '''Commentaar opgave 2
      'Console.WriteLine()
      'Console.WriteLine("Indexer:")
      'Console.WriteLine("{0} {1}", persons(myGuid).FirstName.Value, persons(myGuid).LastName.Value)
    End Sub
  End Class
End Namespace
